### PR TITLE
test(proposals): Add missing test for proposals model

### PR DESCRIPTION
**Pull request by GeekyShacklebolt**
This PR does as follows:

- Add test for model `Proposal`
- Add test for model `ProposalKind`
- Fix flake8 error in common settings

### CODE REVIEW

File: nexus/proposals/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        assert proposal.id
+        assert proposal.speaker == self.user
+        assert proposal.title == 'Test Proposal'
+        assert proposal.kind == self.proposalkind
+        assert str(proposal.level) == 'beginner'
+        assert proposal.duration == '00:30:00'
+        assert proposal.abstract == 'Proposal abstract'
+        assert proposal.description == 'Proposal description'
+        assert proposal.accepted_at is None
+        assert proposal.status == 'submitted'
```
Comment (CuriousLearner) [inline nexus/proposals/tests/test_models.py]: Pick these from the extended choices, don't use the raw value that is there currently.

File: nexus/proposals/tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Standard Stuff
```
Comment (storymode7) [inline nexus/proposals/tests/test_models.py]: Is this required? Since there's no Standard Stuff, I think we can skip this.

Comment (GeekyShacklebolt) [reply nexus/proposals/tests/test_models.py]: Fixed it. Thanks to point out :)

File: nexus/proposals/tests/test_models.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+# nexus Stuff
+from nexus.proposals.models import Proposal, ProposalKind
+
+
+class TestProposalModel(TestCase):
+
+    def setUp(self):
+        self.user = f.create_user(email='test@example.com', password='test')
+        self.proposalkind = f.create_proposal_kind(kind='talk')
```
Comment (CuriousLearner) [inline nexus/proposals/tests/test_models.py]: Please follow python naming conventions.

This should be `proposal_kind`

Comment (GeekyShacklebolt) [reply nexus/proposals/tests/test_models.py]: I'm sorry for the mistake. I have fixed it.

**Pull request merged**